docs/quickstart.md

```diff
@@ -0,0 +1,164 @@
+<!--[metadata]>
++++
+title = "Quickstart Guide"
+description = "Getting started with Docker Compose"
+keywords = ["documentation, docs,  docker, compose, orchestration, containers"]
+[menu.main]
+parent="smn_workw_compose"
+weight=3
++++
+<![end-metadata]-->
+
+
+## Quick start
+
+Let's get started with a walkthrough of getting a simple Python web app running
+on Compose. It assumes a little knowledge of Python, but the concepts
+demonstrated here should be understandable even if you're not familiar with
+Python.
+
+### Installation and set-up
+
+First, [install Docker and Compose](install.md).
+
+Next, you'll want to make a directory for the project:
```

Comment:
Drop the "you'll want to"; we're guiding the reader, so just the instructions for each step; E.g. "create a directory for the project"

I would avoid the "Next,". We're using numbered lists in other parts of the docs; makes it easy for the reader to keep track as well (I left of at step 8)